projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -115,6 +115,19 @@ def download_job(name: str, team: str, path: str):
         cmd.download(team, name, path)
         return f"Downloaded the job with name {name} to {path}. "
 
+    @staticmethod
+    def transform_job(name: str, team: str, path: str):
```

Comment:
this method should not be implemented as part of this PR. The person who is adding the implementation for the operation will add those I think it is better to remove it

Reply:
Removed it.